lib/smart_city/dataset/technical.ex

```diff
@@ -23,7 +23,8 @@ defmodule SmartCity.Dataset.Technical do
           sourceQueryParams: not_required(map()),
           sourceType: not_required(String.t()),
           sourceUrl: String.t(),
-          systemName: String.t()
+          systemName: String.t(),
+          topLevelSelector: not_required(String.t())
```

Comment:
What does this field do?

Reply:
See the linked issue.  This will allow us to define our root element in xml or json. (Something like `ns1:getPublicStationsResponse/stationData` as an example.)